components/ui/autocomplete/src/main/java/mozilla/components/ui/autocomplete/InlineAutocompleteEditText.kt

```diff
@@ -677,6 +677,15 @@ open class InlineAutocompleteEditText @JvmOverloads constructor(
         windowFocusChangeListener?.invoke(hasFocus)
     }
 
+    override fun onTextContextMenuItem(id: Int): Boolean {
+        var newId = id
+        // https://github.com/mozilla-mobile/fenix/issues/1833
```

Comment:
nit: Let's remote the link/comment here. I've filed a separate ticket in this repo: https://github.com/mozilla-mobile/android-components/issues/5512

Would you mind updating the commits message to reference the ticket?
Closes #5512: Strip formatting when pasting text in url bar (or similar)